### PR TITLE
Add TDIGEST.TRIMMED_MEAN

### LINKED ISSUE #2196: Add support for TDIGEST.TRIMMED_MEAN
As of RedisBloom (master) 2.4.0, there is a new command TDIGEST.TRIMMED_MEAN.  We need to add support and the associated tests, as per [this PR](https://github.com/RedisBloom/RedisBloom/commit/15e10a139eb49a1af700d1c2853c32d18186b72b)

### PR DESCRIPTION
**Pull request by bodevone**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `$ tox` pass with this change (including linting)?
- [ ] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Is there an example added to the examples folder (if applicable)?
- [ ] Was the change added to CHANGES file?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change
Add TDIGEST.TRIMMED_MEAN command
Closes #2196

### CODE REVIEW

File: redis/commands/bf/commands.py [hunk 1]
```diff
@@ -416,6 +417,16 @@ def info(self, key):
         """  # noqa
         return self.execute_command(TDIGEST_INFO, key)
 
+    def trimmed_mean(self, key, low_cut_quantile, high_cut_quantile):
```
Comment (dvora-h) [inline redis/commands/bf/commands.py:420]: I think this function need to be in the `TDigestCommands` class

**Review by dvora-h — CHANGES_REQUESTED** [review]

File: redis/commands/bf/commands.py (hunk 1 above)
Comment (bodevone) [reply redis/commands/bf/commands.py:420]: It is already in `TDigestCommands` class

Comment (dvora-h) [reply redis/commands/bf/commands.py:420]: Sorry, my mistake... Ignore it

**Comment by bodevone** [comment]
@dvora-h I am having problem with running test on correct version of BloomFilter (test fails to find command of `TDIGEST.TRIMMED_MEAN`)
Am I doing something wrong?

**Comment by dvora-h** [comment]
@bodevone redismod docker fixed, now it should work.

**Comment by dvora-h** [comment]
@bodevone You need to add this command to MODULE_CALLBACKS (in __init__.py) to convert the response to float.
Let me know if you can do it or you want me to take it.

**Review by dvora-h — APPROVED** [review]

**Comment by dvora-h** [comment]
I hope it's okay that I ended up doing it myself because we wanted to include it in the next version

**Comment by bodevone** [comment]
@dvora-h I am sorry for late response and yes it's totally okay

**Pull request merged**